pkg/network/ovn_kubernetes.go

```diff
@@ -245,6 +245,7 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data.Data["IPsecMachineConfigEnable"] = IPsecMachineConfigEnable
 	data.Data["OVNIPsecDaemonsetEnable"] = OVNIPsecDaemonsetEnable
 	data.Data["OVNIPsecEnable"] = OVNIPsecEnable
+	data.Data["RHEL_OS_ENABLE"] = bootstrapResult.Infra.HostedControlPlane == nil
```

Comment:
that's for hypershift.
you need to check whether there are any nodes with RHEL.